src/test/java/org/opensearch/knn/jni/JNIServiceTests.java

```diff
@@ -466,6 +470,111 @@ public void testCreateIndex_faiss_invalid_invalidIndexDescription() throws IOExc
         );
     }
 
+    @SneakyThrows
+    public void testCreateIndex_faiss_sqfp16_invalidIndexDescription() {
```

Comment:
Could you follow the naming patten? `testX_whenY_thenZ()`

Reply:
Sure, will fix this in next PR